Formula/pypy.rb

```diff
@@ -34,13 +34,13 @@ class Pypy < Formula
 
   resource "bootstrap" do
     on_macos do
-      url "https://downloads.python.org/pypy/pypy2.7-v7.3.4-osx64.tar.bz2"
-      sha256 "ee7bf42ce843596521e02c763408a5164d18f23c9617f1b8e032ce0675686582"
+      url "https://downloads.python.org/pypy/pypy2.7-v7.3.9-osx64.tar.bz2"
+      sha256 "77314f5a6b2cc35d24e6f952bef89f5da612b90e4127a8034aed708d9ae483c4"
```

Comment:
I think we can keep 7.3.4 for bootstrapping (if it works).
It's recommended to bootstrap `pypy` with _another_ version of `pypy` (older one).

Reply:
Understood!